Packs/AutoFocus/Integrations/AutoFocusTagsFeed/README.md

```diff
@@ -0,0 +1,51 @@
+
+## Configure AutoFocusTagsFeed on Cortex XSOAR
```

Comment:
Did you generate this with the sdk command? 
Also, I dont see the "Human Readable Output" under the "autofocus-tags-feed-get-indicators" command.
You need to add the command to the "commands_example" file in order to see the results here.